docs/layer_util.js

```diff
@@ -268,14 +267,26 @@ function showOverlayLayer(overlay, index, map) {
 function resolveOnEeTilesFinished(layerDisplayData, resolve) {
   if (layerDisplayData.tileCallbackId) {
     layerDisplayData.overlay.removeTileCallback(
-        layerDisplayData.tileCallbackId);
+        layerDisplayData.tileCallbackId[0]);
```

Comment:
This method seems to be more than a wrapper now - update doc to reflect?

Reply:
Done, and fixed stale reference in doc. Thanks!